pkg/scheduler/objects/allocation_test.go

```diff
@@ -238,7 +238,7 @@ func TestNewAllocFromSI(t *testing.T) {
 	assert.Assert(t, alloc.IsPlaceholder(), "ask should have been a placeholder")
 	assert.Equal(t, alloc.GetTaskGroup(), "testgroup", "TaskGroupName not set as expected")
 	assert.Equal(t, alloc.GetAsk().GetCreateTime(), time.Unix(past, 0)) //nolint:staticcheck
-	assert.Assert(t, alloc.GetAsk().IsOriginator(), "ask should have been an originator")
+	assert.Assert(t, alloc.IsOriginator(), "ask should have been an originator")
```

Comment:
"ask should not have been an originator"
-> 
"allocation should have been an orignator".

Maybe we can check both allocation & ask's IsOriginator()

Can do the same check for placeholder.